runtime/src/main/kotlin/kotlin/test/Annotation.kt

```diff
@@ -46,7 +46,7 @@ public annotation class AfterEach
  */
 @Retention(AnnotationRetention.SOURCE)
 @Target(AnnotationTarget.CLASS, AnnotationTarget.FUNCTION)
-public annotation class Ignore
+public actual annotation class Ignore
 
-public typealias AfterTest = AfterEach
-public typealias BeforeTest = BeforeEach
+public actual typealias AfterTest = AfterEach
+public actual typealias BeforeTest = BeforeEach
```

Comment:
Do we want to unify names of `BeforeTest`/`AfterTest` annotations by deprecating `BeforeEach`/`AfterEach`?

Reply:
It sounds reasonable.

Reply:
This is a good point. I can rename `After/BeforeEach` into `After/BeforeTest` and both `*Each`' make a typealiases with a deprecation annotation.